tools/experiments/idle.py

```diff
@@ -92,22 +105,28 @@ def kernel_loop(self, loop_index: TInt64):
         self.core.break_realtime()
 
         if differential_mode:
-            for kk in range(len(self.repump_aoms)):
+            for kk in range(len(self.repump_aoms_dds)):
                 # if the repump AOM is off, don't turn on and off the AOM.
```

Comment:
If the repump AOM is off, the user mode should not be able to change to differential mode.